src/routers/datatypeRouters/edges/variants_phenotypes.ts

```diff
@@ -87,20 +233,37 @@ async function variantSearch (input: paramsFormatType): Promise<any[]> {
     delete input.max_alt_freq
     delete input.source
   }
+  let phenotypeFilter = ''
+  if (input.phenotype_id !== undefined) {
+    phenotypeFilter = input.phenotype_id as string
+    delete input.phenotype_id
+  }
+  const hyperEdgeFilters = await getHyperEdgeFilters(input)
```

Comment:
shouldn't need to be preprocessed.

no need to `await`, check comments below for this function